docs/BreakingChanges/076 X509Certificate2.ToString(bool) does not throw now when .NET cannot handle the certificate.md

```diff
@@ -0,0 +1,22 @@
+## 76: X509Certificate2.ToString(bool) does not throw now when .NET cannot handle the certificate
```

Comment:
{
  "Id": "76",
  "Title": "X509Certificate2.ToString(bool) does not throw now when .NET cannot handle the certificate",
  "Details": "Previously, this method would throw if 'true' was passed for the verbose parameter and there were certificates installed that weren't supported by the .Net Framework. Now, the method will succeed and return a valid string that omits the inaccessible portions of the certifiate.",
  "Suggestion": "Any code depending on X509Certificate2.ToString(bool) should be updated to expect that the returned string may exclude some certificate data (such as public key, private key, and extensions) in some cases in which the API would have previously thrown.",
  "Link": "https://msdn.microsoft.com/en-us/library/dn833125(v=vs.110).aspx#Core",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Security.Cryptography.X509Certificates.X509Certificate2.ToString(System.Boolean)"
  ],
  "Related": null,
  "VersionBroken": "4.6",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": true,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
- [x] M:System.Security.Cryptography.X509Certificates.X509Certificate2.ToString(System.Boolean)